tests/unit/select/select_core.js

```diff
@@ -42,6 +42,30 @@
 			"Checkboxes should not have ui-checkbox-on class" );
 	});
 
+	module( "Custom select Multiple Inline" ) ;
+
+	test( "Custom select multiple inline width is adjusted correctly", function() {
+		var selectMenu = $( "#width-test" ),
+			parent = selectMenu.parent(),
+			widths = [ 100, 250, 500, 1000 ],
+			finalResult = 0,
+			result = 0;
+
+		$.each( widths , function( index, width ){
+			parent.css( {"width": width+"px"} );
+			$( "#width-test" )
+				.find( "option" )
+				.prop( "selected", true )
+				.end()
```

Comment:
Remove end and end chain after prop then do `selectMenu.selectmenu( "refresh" );`
It will be much more clear what your doing